src/main/java/ch/njol/skript/expressions/ExprItemAmount.java

```diff
@@ -25,85 +25,68 @@
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
-import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.ExpressionType;
-import ch.njol.skript.lang.SkriptParser;
-import ch.njol.skript.lang.util.SimpleExpression;
-import ch.njol.util.Kleenean;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.util.coll.CollectionUtils;
+
+import java.util.List;
+
 import org.bukkit.event.Event;
 import org.bukkit.inventory.ItemStack;
 import org.eclipse.jdt.annotation.Nullable;
 
-@SuppressWarnings({"unchecked"})
 @Name("Item Amount")
-@Description("Gets the amount of an <a href='classes.html#itemstack'>item stack</a>.")
+@Description("The amount of an <a href='classes.html#itemstack'>item stack</a>.")
 @Examples("send \"You have got %item amount of player's tool% %player's tool% in your hand !\" to player")
-@Since("2.2-dev24")
-public class ExprItemAmount extends SimpleExpression<Number>{
+@Since("2.2-dev24, 2.2-dev35")
```

Comment:
Specify what was added in dev35.

Reply:
he didn't add anything, just an implementation change so this should be removed.